hazelcast-jet-all/src/main/java/com/hazelcast/jet/server/JetCommandLine.java

```diff
@@ -164,9 +165,7 @@ public void run() {
         // top-level command, do nothing
     }
 
-    @Command(description = "Submits a job to the cluster",
-            mixinStandardHelpOptions = true
```

Comment:
are these mixed in by default? so this option was redundant?

Reply:
They are not. But we don't need it for individual commands. We have:
```
jet help COMMAND
```
This mixin will allow also this with the same effect:
```
jet COMMAND -h
```